tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def clear_cache(request: Any) -> None:
+    from app.main import clear_cache
```

Comment:
All imports always should be at the top of a file. Also make sure you have a correct order:
1. Built-in libraries and modules
2. Third-party (installed)
3. Custom